addons/survey/views/survey_question_views.xml

```diff
@@ -221,16 +221,16 @@
                                     <field name='comment_count_as_answer'
                                         attrs="{'invisible': ['|', ('question_type', 'not in', ['simple_choice','multiple_choice', 'matrix']), ('comments_allowed', '=', False)]}"/>
                                 </group>
-                                <group string="Layout">
+                                <group string="Conditional display">
                                     <p class="text-muted" colspan="2" attrs="{'invisible': [('questions_selection', '=', 'all')]}">
                                         Conditional display is not available when questions are randomly picked.
                                     </p>
-                                    <field name="is_conditional" attrs="{'invisible': [('questions_selection', '=', 'random')]}"/>
                                     <field name="allowed_triggering_question_ids" invisible="1"/>
                                     <field name="triggering_answer_ids" widget="many2many_tags" options="{'no_open': True, 'no_create': True}"
                                            domain="[('question_id', 'in', allowed_triggering_question_ids)]"
-                                           attrs="{'invisible': ['|', ('questions_selection', '=', 'random'), ('is_conditional','=', False)],
-                                                   'required': [('is_conditional','=', True)]}">
+                                           attrs="{'invisible': [('questions_selection', '=', 'random')]}"
+                                           placeholder="Optional previous answers required"
```

Comment:
I found this placeholer a bit strange when testing, as placeholder is usually used as an example of what it will contain rather that a "tip / constraint".
But if you and François are ok with it, I will still sleep at night if it stays ^^